packages/plugin-essentials/sources/commands/remove.ts

```diff
@@ -89,11 +89,16 @@ export default class RemoveCommand extends BaseCommand {
 
         for (const target of targets) {
           const descriptors = workspace.manifest.getForScope(target);
-          const stringifiedIdents = [...descriptors.values()].map(structUtils.stringifyIdent);
+          const stringifiedIdents = [...descriptors.values()].map(stringifiedIdent => {
```

Comment:
Technically, they're not `stringifiedIdent`s, but `descriptor`s we stringify. (I agree with avoiding the implicit callback, it was a mistake on my part.)

Reply:
Woops my bad! Wouldn't make sense to stringify a stringified ident indeed 😄